app/templates/components/notification-dropdown.hbs

```diff
@@ -4,7 +4,7 @@
 <div class="ui wide notification popup bottom left transition ">
   <div class="ui basic inverted horizontal segments">
     <div class="ui basic left aligned segment weight-800">
-      <p>{{t 'Notifications'}}</p>
+      <a href="/notifications">{{t 'Notifications'}}</a>
```

Comment:
Please use `link-to` helper here. https://guides.emberjs.com/v2.5.0/templates/links/